File: sample/src/main/java/com/efraespada/stringobfuscator/MainActivity.java
package com.efraespada.stringobfuscator;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.TextView;

import com.stringcare.library.SC;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        SC.init(getApplicationContext());
        // SC.initForLib(getApplicationContext(), this);

        int stringId = R.string.hello;

        String message = getString(stringId);
        message += " is ";
        message += SC.getString(stringId);

        // secret var
        String mySecret = "lalilulelo";

        message += "\n\nFor Metal Gear lovers:\n\n\"Snake, the password is " + SC.encryptString(message)
            + "\n\n.. or " + SC.decryptString(SC.encryptString(mySecret)) + "\"";

        ((TextView) findViewById(R.id.example_a)).setText(message);

        String numbers = getString(R.string.test_a, "hi", 3) + " is " + SC.getString(R.string.test_a, "hi", 3);
        ((TextView) findViewById(R.id.example_b)).setText(numbers);

    }
}

File: app/src/main/java/com/stringcareapp/scsample/MainActivity.kt
package com.stringcareapp.scsample

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.stringcare.library.SC
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        SC.initOnModule(applicationContext, BuildConfig::class.java)

        val stringId = R.string.hello

        var message = getString(stringId)
        message += " is "
        message += SC.getString(stringId)

        // secret var
        val mySecret = "lalilulelo"

        message += ("\n\nFor Metal Gear lovers:\n\n\"Snake, the password is " + SC.encryptString(message)
                + "\n\n.. or " + SC.decryptString(SC.encryptString(mySecret)) + "\"")

        example_a.text = message
    }

}
